src/join/join.rs

```diff
@@ -36,6 +36,13 @@ enum Sep {
     Whitespaces,
 }
 
+#[derive(Copy, Clone, PartialEq)]
+enum CheckOrder {
+    Default,
```

Comment:
Shouldn’t the default just be Disabled? I’m pretty sure that’s what it is for the GNU version.

Reply:
The default behavior is neither enabled nor disabled. The warning should still be shown by default, but the program should not exit early when the input lines are not ordered.

Reply:
Ah alright, I didn't see that part of the documentation at first.